CHANGELOG.md

```diff
@@ -516,6 +516,13 @@ Other minor additions
   index-∈-lookup : (i : Fin n) (xs : Vec A n) → Any.index (∈-lookup i xs) ≡ i
   ```
 
+* Added new proofs to `Data.Tree.Binary.Properties`:
+  ```agda
+  map-compose : ∀ {f₁ : N₁ → N₂} {f₂ : N → N₁} {g₁ : L₁ → L₂} {g₂ : L → L₁} →
```

Comment:
We typically remove all of the implicit arguments in the CHANGELOG
to focus on the interesting part of the type.